recipes/py-exe-builder/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "py-exe-builder" %}
+{% set version = "0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/py-exe-builder-{{ version }}.tar.gz
+  sha256: 9bb355b1bf288250028387f778e1d3508917c410b0501ff5dc422e3c1f174059
+
+build:
+  skip: true  # [not win]
```

Why not make this `noarch`?

```suggestion
  noarch: python
```

---

> Why not make this `noarch`?

The package is indeed a noarch on Pypi, but you won't be able to install it on another OS than Windows. It's a pure python package, but it relies on Py2Exe, wich is a Windows only package made to convert py files to exe.